examples/diffusion/laplace_iga_interactive.py

```diff
@@ -8,13 +8,31 @@
 built-in CAD routines, which is then saved in custom HDF5-based files with
 .iga extension.
 
+Usage Examples
+--------------
+
+Default options, storing results in this file's parent directory::
+
+  $ python3 laplace_iga_interactive.py
```

Comment:
I missed this: the other examples use the path relative to the main sfepy directory, i.e. use `examples/diffusion/laplace_iga_interactive.py` here.